src/store/migrations.test.js

```diff
@@ -72,6 +75,12 @@ describe('Redux migrations', () => {
 
       expect(actualOutput).toEqual(expectedOutput);
     });
+
+    it('builds without crashing', () => {
+      let state = rootReducer(undefined, { type: FAKE_ACTION });
```

Comment:
maybe extract this to a function `getInitialState()` with a comment saying that because the action doesn't exist, all the reducers will just fall through and return their initialState. I had to think about it ;)

Reply:
Yeah, good call :+1: